docs/source/en/guides/cli.md

```diff
@@ -0,0 +1,445 @@
+```
+
+### Quiet mode
+
+By default, the `huggingface-cli download` command will be verbose. It will print details such as warning messages, information about the downloaded files, and progress bars. If you want to silence all of this, use the `--quiet` option. Only the last line (i.e. the path to the downloaded files) is printed. This can prove useful if you want to pass the output to another command in a script.
+
+```bash
+>>> huggingface-cli download gpt2 --quiet
+/home/wauplin/.cache/huggingface/hub/models--gpt2/snapshots/11c5a3d5811f50298f278a704980280950aedb10
+```
+
+## huggingface-cli upload
+
+Use the `huggingface-cli upload` command from the terminal to directly upload files to the Hub. Internally it uses the same [`upload_file`] and [`upload_folder`] helpers described in the [Upload](./upload) guide. In the examples below, we will walk through the most common use cases. For a full list of available options, you can run:
+
+```bash
+>>> huggingface-cli upload --help
+```
+
+### Upload an entire folder
+
+The default usage for this command is:
+
+```bash
+# Usage:  huggingface-cli upload [repo_id] [local_path] [path_in_repo]
+```
+
+To upload the current directory at the root of the repo, use:
+
+```bash
+>>> huggingface-cli my-cool-model . .
+https://huggingface.co/Wauplin/my-cool-model/tree/main/
+```
+
+<Tip>
+
+If the repo doesn't exist yet, it will be created automatically.
+
+</Tip>
+
+You can also upload a specific folder:
+
+```bash
+>>> huggingface-cli my-cool-model ./models .
+https://huggingface.co/Wauplin/my-cool-model/tree/main/
+```
+
+Finally, you can upload a folder to a specific destination on the repo:
+
+```bash
+>>> huggingface-cli my-cool-model ./path/to/curated/data /data/train
+https://huggingface.co/Wauplin/my-cool-model/tree/main/data/train
+```
+
+### Upload a single file
+
+We saw above how to upload an entire folder. You can also upload a single file by setting `local_path` to point to a file on your machine. If that's the case, `path_in_repo` is optional and will default to the name of your local file:
+
+```bash
+>>> huggingface-cli upload Wauplin/my-cool-model ./models/model.safetensors
+https://huggingface.co/Wauplin/my-cool-model/blob/main/model.safetensors
+```
+
+If you want to upload a single file to a specific directory, set `path_in_repo` accordingly:
+
+```bash
+>>> huggingface-cli upload Wauplin/my-cool-model ./models/model.safetensors /vae/model.safetensors
+https://huggingface.co/Wauplin/my-cool-model/blob/main/vae/model.safetensors
+```
+
+### Upload multiple files
+
+To upload multiple files from a folder at once without uploading the entire folder, use the `--include` and `--exclude` patterns. It can also be combined with the `--delete` option to delete files on the repo while uploading new ones. In the example below, we sync the local Space by deleting remote files and uploading all files, except the ones in `/logs` and 
```

Comment:
```suggestion
To upload multiple files from a folder at once without uploading the entire folder, use the `--include` and `--exclude` patterns. It can also be combined with the `--delete` option to delete files on the repo while uploading new ones. In the example below, we sync the local Space by deleting remote files and uploading all files except the ones in `/logs`: 
```